packages/pages/src/common/src/parsers/templateParser.test.ts

```diff
@@ -68,10 +68,83 @@ describe("makeClientTemplate", () => {
       `import { Template, TemplateRenderProps } from "@yext/pages";`
     );
   });
+
+  it("does not return tsdoc comments", () => {
+    const sourceParser = createParser(
+      `/**
+      * mock tsdoc comments
+      */
+      const foo = 5;
+      export default foo`
+    );
+    const testParser = createParser(``);
+    const templateParser = new TemplateParser(
+      sourceParser
+    ).makeClientTemplateFromSfp(testParser);
+    expect(templateParser.fileContents).not.toContain(` mock tsdoc comments`);
+  });
+});
+
+describe("complex makeClientTemplate usages", () => {
+  it("correctly returns expected file contents", () => {
+    const sourceParser = new SourceFileParser(
+      "./tests/fixtures/sourceFileTemplates/static.tsx",
```

Comment:
Add a test for an entity template as well that also uses a function in the config that comes from another file. Something like:

```
import { dedupe } from 'foo';
export const config = {
  .......
 fields: dedupe(["id", "id", "name"])
}
```

Reply:
Sounds good. Does import { isProduction } from "@yext/pages/util"  work in this case? Just added the state.tsx template with isProduction being called in the default export.